packages/monitoring/src/main/HealthCheckMiddleware.ts

```diff
@@ -0,0 +1,70 @@
+import { Request, Response, NextFunction } from 'express'
+import HealthCheck from './HealthCheck'
+import { HealthCheckOptions } from './types/HealthCheckOptions'
+import DeploymentInfo from './DeploymentInfo'
+
+/**
+ * HealthCheckMiddleware class provides middleware functions for handling
+ * health and deployment information endpoints in an Express.js application.
+ *
+ * Example:
+ *
+ * ```typescript
+ * const healthMiddleware = new HealthCheckMiddleware(options);
+ * app.get('/health', healthMiddleware.health);
+ * app.get('/info', healthMiddleware.info);
+ * app.get('/ping', healthMiddleware.ping);
+ * ```
+ */
+export default class HealthCheckMiddleware {
```

Comment:
meh... could call this MonitoringMiddleware?

Reply:
I agree, resolved in https://github.com/ministryofjustice/hmpps-typescript-lib/pull/7/commits/e4f6e7f0927875a8e8a08521d93263b19623c3d1